internal/expr/parts.go

```diff
@@ -10,7 +14,7 @@ type queryPart interface {
 	String() string
 
 	// ToSQL returns the SQL representation of the part.
-	toSQL() string
+	toSQL([]string, int) string
```

Comment:
As we discussed during the call, you're right that this is a bad smell. Let's just kill this method and move that trivial logic to the call site, and implement a private `part()` marker method that does the validation work.

Reply:
I changed this a few days ago, forgot to reply here!